src/xcode/ENA/ENA/Source/Views/ExposureSubmission/DynamicLegalExtendedCell.xib

```diff
@@ -30,49 +30,57 @@
+                                            </label>
+                                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" distribution="fillProportionally" spacing="20" translatesAutoresizingMaskIntoConstraints="NO" id="8Pe-9Z-Ydm" userLabel="Content Stack View 1">
+                                                <rect key="frame" x="0.0" y="40.5" width="248" height="184"/>
+                                            </stackView>
+                                        </subviews>
+                                    </stackView>
+                                    <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" distribution="fillProportionally" spacing="20" translatesAutoresizingMaskIntoConstraints="NO" id="711-w9-VbR">
+                                        <rect key="frame" x="0.0" y="244.5" width="248" height="553"/>
+                                        <subviews>
+                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" verticalCompressionResistancePriority="751" text="Label" textAlignment="natural" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="79Y-9g-GkT" userLabel="Description Label 2" customClass="ENALabel" customModule="ENA" customModuleProvider="target">
+                                                <rect key="frame" x="0.0" y="0.0" width="248" height="20.5"/>
+                                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                                <nil key="textColor"/>
+                                                <nil key="highlightedColor"/>
+                                                <userDefinedRuntimeAttributes>
+                                                    <userDefinedRuntimeAttribute type="string" keyPath="ibEnaStyle" value="headline"/>
+                                                </userDefinedRuntimeAttributes>
+                                            </label>
+                                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="20" translatesAutoresizingMaskIntoConstraints="NO" id="3ck-WL-Nhf" userLabel="Content Stack View 2">
+                                                <rect key="frame" x="0.0" y="40.5" width="248" height="512.5"/>
+                                            </stackView>
+                                        </subviews>
+                                    </stackView>
+                                </subviews>
+                                <constraints>
+                                    <constraint firstItem="711-w9-VbR" firstAttribute="top" secondItem="INJ-xI-Je9" secondAttribute="bottom" constant="20" id="yiw-As-gz2"/>
```

Comment:
I'm not sure if this constraint has any effect. I think, maybe, it can be removed:

<img width="430" alt="Bildschirmfoto 2020-12-22 um 15 41 45" src="https://user-images.githubusercontent.com/1941000/102900353-73822d80-446c-11eb-9ca4-96a54eb20a6f.png">

Reply:
Good point – removed it.